app/serializers/disability_claim_base_serializer.rb

```diff
@@ -30,4 +30,39 @@ def open
   def waiver_submitted
     object.data['waiver5103Submitted']
   end
+
+  protected
+
+  def with_object_data(*keys)
+    val = object.data.dig(*keys)
+    if val.present?
+      yield val
+    end
+  end
+
+  def bool_from_yes_no(*keys)
+    with_object_data(*keys) do |s|
+      case s.downcase
+      when 'yes' then true
+      when 'no' then false
+      else nil
+        # Todo: Log weird input
+      end
+    end
+  end
+
+  def date_from_string(*keys, format: '%m/%d/%Y')
+    with_object_data(*keys) do |s|
+      Date.strptime(s, format)
+    end
+  end
+
+  # For date-times recording a computer event and therefore known to the
+  # second EVSS uses a UNIX timestamp in milliseconds. They are mostly
+  # in intent-to-file and documents.
+  def date_from_unix_milliseconds(*keys)
```

Comment:
Unix times are used for keys I haven't added to the updates list yet. `vbaDocuments` `receiptDate`, `uploadDate`, `allIntentToFiles` `receivedDate`. It might end up being YAGNI and cut. Leaving it for now.